frontend/src/stores/OutputStore.js

```diff
@@ -316,6 +325,22 @@ class OutputStore {
             return dataset.metadata.name;
         }
     }
+
+    @action.bound saveUserPlotSettings(e) {
+        // set user configurable plot settings
+        if (_.has(e, "xaxis.range[0]") && _.has(e, "xaxis.range[1]")) {
+            this.userPlotSettings["xaxis"] = [e["xaxis.range[0]"], e["xaxis.range[1]"]];
+        }
+        if (_.has(e, "yaxis.range[0]") && _.has(e, "yaxis.range[1]")) {
+            this.userPlotSettings["yaxis"] = [e["yaxis.range[0]"], e["yaxis.range[1]"]];
+        }
+        if (_.has(e, "yaxis.autorange")) {
```

Comment:
things got weird if you reset on plotly; i had to handle that case. Also, in some cases, only xaxis or yaxis values were set and not all. Please check all edge cases.